src/content/documentation/develop/best-practices-for-collecting-user-data-consents.md

```diff
@@ -148,17 +148,15 @@ Before designing your consent dialogs, you should determine the privacy consent
 In this example:
 
 1. Your extension should start up with all data collection functionality disabled.
-2. Shortly after installation (see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/)), present them with a consent dialog.
+2. Immediately after installation (see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/)), present them with a consent dialog.
    On this dialog:
 3. The option for collecting personally identifying information must default to disabled so that the user has to actively opt-in.
 4. The options for technical data may default to enabled, so that the user may choose to opt-out.
 5. If the user:
     - opts IN to all features &rarr; continue with all extension features enabled.
-    - opts OUT of any feature that cannot be disabled &rarr; offer them the option to remove the extension. If they choose not to remove the extension present the consent dialog again.
+    - opts OUT of any feature that cannot be disabled &rarr; uninstall the extension with [`management.uninstallSelf()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/management/uninstallSelf).
```

Comment:
This doesn't sound right. How can you opt out of something that cannot disabled?

Reply:
I see, and yes that's fair. It could be made more explicit from the perspective of what a user can do.

I'll make it to be something like, if that sounds more in line with what we mean.

```
If the user:
- consents to all features ...
- doesn't consent to any feature that cannot be disabled ...
- doesn't consent to any feature that can be disabled ...
```

Any thoughts? I'll update diagram accordingly.

Reply:
@wagnerand made the changes (to both diagram and accompanying description text) to sound more explicit from perspective of user action performed, and used "data collection" instead of "features" to be more explicit. Please take a look! Thanks